templates/data-providers/styles.module.css

```diff
@@ -15,3 +15,7 @@
     margin-left: 1rem;
   }
 }
+
+.form {
+  margin-top: 1rem;
```

Comment:
Is it a problem of the headline spacing?

Reply:
yes, the margin is taken from [card title](https://github.com/oacore/design/blob/master/src/elements/card/card.css#L20) and it didn't look great so I extended it a little bit.